src/services/usersService.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../dataBase');
+const { v4: uuidv4 } = require('uuid');
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.STRING,
+    primaryKey: true,
+    allowNull: false,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+}, {
+  tableName: 'users',
+  updatedAt: false,
+  createdAt: false,
+});
```

Comment:
i think you shoul move to a different file

Reply:
> i think you shoul move to a different file

The same comment as above